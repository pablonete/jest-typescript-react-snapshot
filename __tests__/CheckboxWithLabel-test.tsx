import * as React from 'react';
import * as renderer from 'react-test-renderer';
import { CheckboxWithLabel } from "../CheckboxWithLabel";

it('CheckboxWithLabel changes the text after click', () => {
  const checkbox = renderer.create(
    <CheckboxWithLabel labelOn="On" labelOff="Off" />
  );

  expect(checkbox.toJSON()).toMatchSnapshot();
});
