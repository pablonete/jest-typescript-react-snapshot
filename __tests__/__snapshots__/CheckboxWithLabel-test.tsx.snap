exports[`test CheckboxWithLabel changes the text after click 1`] = `
<label>
  <input
    checked={false}
    onChange={[Function]}
    type="checkbox" />
  Off
</label>
`;
